sdk/python/feast/feature_server.py

```diff
@@ -307,8 +307,7 @@ def start_server(
     keep_alive_timeout: int,
     registry_ttl_sec: int,
 ):
-    # TODO RBAC remove and use the auth section of the feature store config instead
-    auth_manager_type = auth_manager_type_from_env()
+    auth_manager_type = str_to_auth_manager_type(store.config.auth_config.type)
```

Comment:
why is this needed?

Reply:
because underlying layers expect an enum, not a str

Reply:
changes as per [comments](https://github.com/RHEcosystemAppEng/feast/pull/37#discussion_r1672191895) need to pass AuthManagerType instead of string cc : @dmartinol